content/en/platform/corda/4.13/enterprise/node/ledger-recovery/automatic-ledger-recovery.md

```diff
@@ -0,0 +1,46 @@
+---
+date: '2023-11-20'
+menu:
+  corda-enterprise-4-13:
+    identifier: corda-enterprise-4-13-corda-ledger-recovery-automatic
+    name: "Automatic ledger recovery"
+    parent: corda-enterprise-4-13-corda-ledger-recovery
+tags:
+- ledger recovery
+
+title: Automatic ledger recovery
+weight: 900
+---
+
+# Automatic ledger recovery
+
+[Ledger recovery]({{< relref "ledger-recovery.md" >}}) was introduced to complement normal backup and recovery process. Ledger recovery involves using [the Ledger Recovery flow]({{< relref "ledger-recovery-flow.md" >}}) to enable a node to restore transactions from its peer nodes in the event of local data loss.
+
```

Comment:
I know i mentioned we are not supporting the general system flow. But i guess as configuration deals in system flows we need to introduce the term.
Maybe describe something like a system flow will run automatically startup and the only system flow supported is EnterpriseLedgerRecoveryFlow

Need also explain the concepts of supercedes, incase default not good enough for user. i.e. another flow can supercede the recoveryflow